content/docs/command-reference/status.md

```diff
@@ -43,11 +43,13 @@ paths to tracked files or directories (including paths inside tracked
 directories), `.dvc` files, and stage names (found in `dvc.yaml`).
 
 If no differences are detected, `dvc status` prints
-`Data and pipelines are up to date.` If differences are detected by
-`dvc status`, the command output indicates the changes. For each stage with
-differences, the changes in <abbr>dependencies</abbr> and/or
-<abbr>outputs</abbr> that differ are listed. For each item listed, either the
-file name or hash is shown, along with a _state description_, as detailed below:
+`Data and pipelines are up to date.` or
+`Cache and remote '{remote}' are in sync.` for `dvc status -c/-r`. If
```

Comment:
Hmmm. I don't see this in https://github.com/iterative/dvc.org/blame/master/content/docs/command-reference/status.md#L45 or https://dvc.org/doc/command-reference/status 🤔

Reply:
I don't understand what happened since the blame doesn't even register this merge. But in any case, I further reviewed this and re-applied it in https://github.com/iterative/dvc.org/pull/1881/commits/246575a4a93f2e1a7cde85847f8084238741a9f8 (part of #1881).